src/components/tracked-controls.js

```diff
@@ -130,14 +130,16 @@ module.exports.Component = registerComponent('tracked-controls', {
     var buttonStates = this.buttonStates;
     var evtName;
     var previousButtonState = buttonStates[id] = buttonStates[id] || {};
-    if (buttonState.pressed === previousButtonState.pressed) { return; }
+    if (buttonState.pressed === previousButtonState.pressed) { return false; }
     if (buttonState.pressed) {
       evtName = 'down';
     } else {
       evtName = 'up';
     }
     this.el.emit('button' + evtName, {id: id});
     previousButtonState.pressed = buttonState.pressed;
+    // return true so handleButton knows something changed
```

Comment:
Yes but in the method doc so a reader can understand what the method returns when reading at the function doc. [Example](https://github.com/aframevr/aframe/blob/master/src/components/material.js#L130)